docs/assets/style.css

```diff
@@ -0,0 +1,13 @@
+html,
+body {
+  min-height: 100vh;
+}
```

Comment:
could we just use min-vh-100 in the HTML ?

Reply:
This is an optional comment. As there is a style.css file anyway, no problem to keep thes rules